book/master_middleman/source/subnavs/apache-hawq-nav.erb

```diff
@@ -117,6 +117,9 @@
           <li>
             <a href="/docs/userguide/2.2.0.0-incubating/admin/logfiles.html">HAWQ Administrative Log Files</a>
           </li>
+          <li>
+            <a href="/docs/userguide/2.2.0.0-incubating/admin/packcore.html">HAWQ packcore utility</a>
```

Comment:
this is very minor - how about capitalizing the U in "utility"?

Reply:
@lisakowen changed the heading format to make it similar to naming convention of other headings.

Reply:
Thanks @outofmem0ry - merging this in now.